src/core/mantle/evaluator/simple/expression/variableReplacer.ts

```diff
@@ -95,3 +96,23 @@ function containsValidJSONPath(expression: string, feature: Feature): boolean {
     return false;
   }
 }
+
+const makeReservedWord = (str: string) => `$reearth_${str}_$`;
```

Comment:
Can you use some other special character instead of `$` as it might be used somewhere else?. How about `#`?
```suggestion
const makeReservedWord = (str: string) => `#reearth_${str}_#`;
```

Reply:
`#` seems to be reserved word, so I will keep using `$`:pray: